### ISSUE TITLE
Chunked Upload fails on space characters in file name

### ISSUE THREAD

**Issue by petzlux** [opendatateam/udata#1695]
@noirbizarre  

I have tested our recent upload problems at https://data.public.lu/ , and discovered one possible issue with the filename to be uploaded. 
For an equivalent file of 50mb, the upload fails for `test (1).zip` but works for `test(1).zip`  . Can anyone reproduce this ?

**Comment by noirbizarre** [comment]
Interesting :thinking: 
I'll take a closer look: it might be possible that spaces are not preperly handled, either in udata or in `flask-fs`

**Comment by petzlux** [comment]
So can you reproduce the problem ? Just want to make sure it's not something to do with our Fuse to S3 setup ?

**Comment by noirbizarre** [comment]
No, I tried locally and a deployed instance, I'm unable to reproduce the issue. Spaces seems to be handled just fine.
I tried with small and medium files.

I'll do more test tomorrow

**Comment by petzlux** [comment]
https://github.com/opendatalu/gouvlu/issues/87
